### DISCUSSION TITLE
How long after updating?

### DISCUSSION THREAD

**Discussion by codebydean** [anuraghazra/github-readme-stats#2687] [Q&A] [answered]
How long does it usually take for the stats to be updated?

Because sometimes when I do a commit in my own public repo, it does not increment
![image](https://user-images.githubusercontent.com/71401702/236328322-7cb5434f-2d0b-4a53-8e79-3eed78190489.png)

Thanks in advance

**Comment by qwerty541** [comment, marked as answer]
It depends from `cache_seconds` option. By default it was set to 14400 seconds ( 4 hours ). You can set custom value from 14400 to 86400 seconds. See: https://github.com/anuraghazra/github-readme-stats#common-options

**Reply by codebydean** [reply]
Thank you very much!
